config/initializers/menus.rb

```diff
@@ -337,10 +337,16 @@
 
   menu.push :admin_projects_settings,
             { controller: "/admin/settings/project_custom_fields", action: :index },
-            if: Proc.new { User.current.admin? },
+            if: Proc.new { User.current.admin? && OpenProject::FeatureDecisions.stages_and_gates_active? },
```

Comment:
This is at the wrong menu item, @toy. It should be at `:project_life_cycle_step_definitions_settings`

Reply:
Indeed, thank you, I changed this menu item to also lead to life cycle and apparently changed the wrong condition before rolling back the change